### PR TITLE
Payments

### PR DESCRIPTION
**Pull request by matthopson**
Add methods for new payments apiv3 endpoints.

### CODE REVIEW

**Comment by semmons99** [comment]
any tests for this?

**Comment by matthopson** [comment]
I suppose that would be the responsible thing to do. Thanks!

**Comment by matthopson** [comment]
@semmons99 tests added.
@kristenboydstun have one more glance if you don't mind.

**Comment by semmons99** [comment]
Looks good to me.

File: test/paymentNotes.coffee [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  it 'should be able to load all payment notes for a team', (done) ->
+    teamsnap.loadPaymentNotes team.id, (err, result) ->
+      expect(err).to.be.null
+      result.should.be.an('array')
+      done()
+
+  it 'should be able to create a payment note', (done) ->
+    paymentNote = teamsnap.createPaymentNote()
+    paymentNote.teamId = team.id
+    paymentNote.memberPaymentId = 1
```
Comment (kristenboydstun) [inline test/paymentNotes.coffee]: Does this mean member payment with id 1 needs to exist already? If so, where was it created?

**Comment by kristenboydstun** [comment]
:ship:

**Pull request merged**